Source/WebInspectorUI/UserInterface/Views/DOMTreeContentView.js

```diff
@@ -694,6 +702,97 @@ WI.DOMTreeContentView = class DOMTreeContentView extends WI.ContentView
+            return;
+        }
+
+        let contentElement = document.createElement("div");
+        contentElement.classList.add("user-preferences-content");
+
+        let createSelectElement = (label, preference, values = [], defaultValue) => {
+            if (!preference || !values.length)
+                return;
+
+            let labelElement = contentElement.appendChild(document.createElement("label"));
+            let labelTextElement = labelElement.appendChild(document.createElement("span"));
+            labelTextElement.textContent = label;
+
+            let selectElement = labelElement.appendChild(document.createElement("select"));
+            selectElement.setAttribute("name", preference);
+
+            for (let value of values) {
+                let optionElement = selectElement.appendChild(document.createElement("option"));
+                optionElement.value = value;
+                optionElement.textContent = this._userPreferenceValueLabel(value);
+
+                let selectedValue = this._overridenUserPreferences.get(preference) || defaultValue;
+
+                if (value === selectedValue)
+                    optionElement.setAttribute("selected", true);
+            }
+
+            selectElement.addEventListener("change", (event) => {
+                let value = event.target.value;
+
+                WI.cssManager.overrideUserPreference(preference, value);
+
+                if (value !== defaultValue)
+                    this._overridenUserPreferences.set(preference, value);
+                else
+                    this._overridenUserPreferences.delete(preference);
+
+                this._overrideUserPreferencesNavigationItem.activated = this._overridenUserPreferences.size > 0;
+            });
+
+            return selectElement;
+        }
+
+        let calculateTargetFrame = () => {
+            return WI.Rect.rectFromClientRect(this._overrideUserPreferencesNavigationItem.element.getBoundingClientRect()).pad(2);
+        }
+
+        let preferredEdges = [WI.RectEdge.MAX_Y, WI.RectEdge.MAX_X];
+
+        this._userPreferencesOverridesPopover = new WI.Popover(this);
+        this._userPreferencesOverridesPopover.windowResizeHandler = () => {
+            this._userPreferencesOverridesPopover.present(calculateTargetFrame(), preferredEdges);
+        };
+
+        let accessibilityHeader = contentElement.appendChild(document.createElement("h5"));
+        accessibilityHeader.textContent = WI.UIString("Accessibility Preferences", "Accessibility Preferences @ User Preferences Overrides", "Header for section with accessibility user preferences.");
+
+        let userPreferenceValues = [InspectorBackend.Enum.Page.UserPreferenceValue.System, InspectorBackend.Enum.Page.UserPreferenceValue.On, InspectorBackend.Enum.Page.UserPreferenceValue.Off];
+
+        // COMPATIBILITY (macOS 13.0, iOS 16.0): `PrefersReducedMotion` value for `Page.UserPreference` did not exist yet.
```

Comment:
we should be able to assume this since we can only reach this code if this is already true (i.e. `_overrideUserPreferencesNavigationItem` is only clickable if the protocol supports this)

please either remove these checks or if you really wanna be safe add some sort of check on :777 that bails if the `contentElement` is empty